intro/journaling_api.md

```diff
@@ -91,7 +93,8 @@ For example:
 curl -X GET \
   https://api.adobe.io/events/organizations/23294/integrations/54108/f89067f2-0d50-4bb2-bf78-209d0eacb6eb \
   -H "Authorization: Bearer $USER_TOKEN" \
-  -H "x-api-key: $API_KEY"
+  -H "x-api-key: $API_KEY" \
+  -H "x-ims-org-id: 4CC7D9704674CFB2138A2C54@AdobeOrg"
```

Comment:
My apologies for a pure bikeshed comment - 

In its current form, the `org id` is sticking out aesthetically. How do these options look? - 

1. 
```
https://api.adobe.io/events/organizations/23294/integrations/54108/f89067f2-0d50-4bb2-bf78-209d0eacb6eb?since=moose:e7ba778b-dace-4994-96e7-da80e7125233:2159b72c-e284-4899-b572-08da250e3614 \
  -H "x-ims-org-id: 4CC7D9704674CFB2138A2C54@AdobeOrg" \
  -H "Authorization: Bearer $USER_TOKEN" \
  -H "x-api-key: $API_KEY"
```

```
https://api.adobe.io/events/organizations/23294/integrations/54108/f89067f2-0d50-4bb2-bf78-209d0eacb6eb?since=moose:e7ba778b-dace-4994-96e7-da80e7125233:2159b72c-e284-4899-b572-08da250e3614 \
  -H "Authorization: Bearer $USER_TOKEN" \
  -H "x-api-key: $API_KEY" \
  -H "x-ims-org-id: $ORG_ID"
```

Reply:
The **first** option looks better as the user would be able to **connect** with it. If we have a consensus on that then I will raise a PR for the same.
@rperelma @theManikJindal WDYT?

Reply:
I am fine with either of the options. I think you can go ahead and amend your PR. I'll approve.